app/gui/view/graph-editor/src/component/visualization/foreign/java_script/binding.rs

```diff
@@ -97,21 +97,33 @@ pub struct JsTheme {
     styles: StyleWatch,
 }
 
-/// A color in RGBA representation. Can be passed to JavaScript. Implements the `From` trait for
-/// convertion from `color::Lcha`.
-#[wasm_bindgen]
-#[derive(Debug, Copy, Clone)]
-pub struct JsColor {
-    /// The red part as a float between 0 and 1
-    pub red:   f32,
-    /// The green part as a float between 0 and 1
-    pub green: f32,
-    /// The blue part as a float between 0 and 1
-    pub blue:  f32,
-    /// The opacity as a float between 0 and 1
-    pub alpha: f32,
+
+// The code generated by #[wasm_bindgen] explicitly calls drop, even on Copy types.
+// As the generated code appears outside the type definition, we cannot attach this exception
+// to the type itself. Thus, we place the type in a special module and allow the lint there.
+#[allow(clippy::drop_copy)]
+#[allow(missing_docs)] // This is a faux module, it is not meant to be documented.
+mod js_color {
+    use super::*;
+    /// A color in RGBA representation. Can be passed to JavaScript. Implements the `From` trait for
+    /// convertion from `color::Lcha`.
+    #[wasm_bindgen]
+    #[derive(Debug, Copy, Clone)]
+    pub struct JsColor {
+        /// The red part as a float between 0 and 1
+        pub red:   f32,
+        /// The green part as a float between 0 and 1
+        pub green: f32,
+        /// The blue part as a float between 0 and 1
+        pub blue:  f32,
+        /// The opacity as a float between 0 and 1
+        pub alpha: f32,
+    }
 }
 
+// Make it as if module `js_color` members were defined here.
+pub use js_color::*;
+
```

Comment:
Is this comment needed? I think the code is self-explanatory.

Reply:
Ok, wwill remove.